app/views/tcsp/summary.scala.html

```diff
@@ -107,18 +123,28 @@
         }
 
         @model.servicesOfAnotherTCSP.map { s =>
-
             @checkYourAnswersRow(
                 question = Messages("tcsp.anothertcspsupervision.title"),
                 editLinkTag = "anothertcspsupervision-edit",
                 editUrl = controllers.tcsp.routes.AnotherTCSPSupervisionController.get(true).toString
             ) {
-                    @s match {
-                        case ServicesOfAnotherTCSPNo => { @Messages("lbl.no") }
-                        case ServicesOfAnotherTCSPYes(v) => { @Messages("tcsp.cya.mlrregno"): @v }
-                    }
+                @s match {
+                    case ServicesOfAnotherTCSPNo => { @Messages("lbl.no") }
+                    case ServicesOfAnotherTCSPYes(v) => { @Messages("lbl.yes") }
+                }
             }
+        }
 
+        @model.servicesOfAnotherTCSP.map {
+            case ServicesOfAnotherTCSPYes(v) => {
```

Comment:
this won't fail if case is ServicesOfAnotherTCSPNo?

Reply:
Yes it did, fixed.